src/pages/index.js

```diff
@@ -6,6 +6,60 @@ import PopupWithImage from "../components/PopupWithImage.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import UserInfo from "../components/UserInfo.js";
 import "../pages/index.css";
+import Api from "../components/Api.js";
+import PopupConfirm from "../components/PopupConfirm.js";
+import { data } from "autoprefixer";
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "d0764cda-b1ee-4837-a694-3cacde791a83",
+    "Content-Type": "application/json",
+  },
+});
+
+api
+  .getInitialCards()
+  .then((res) => {
+    cardSection.renderItems(res);
+  })
+  .catch((err) => {
+    console.error(err);
+  });
+
+api
+  .getUserInfo()
+  .then((res) => {
+    userInformation.setUserInfo({
+      name: res.name,
+      description: res.about,
+      avatar: res.avatar,
+    });
+  })
+  .catch((err) => {
+    console.error(err);
+  });
+
+const deleteCardModal = new PopupConfirm(
+  "#modal-delete-card",
+  handleDeleteClick
+);
+deleteCardModal.setEventListeners();
+
+function handleDeleteClick(card) {
+  deleteCardModal.open();
+  deleteCardModal.setconfirmCall(() => {
+    api
+      .deleteCard(card.getId())
+      .then(() => {
+        card._handleDeleteCard();
+      })
+      .catch((err) => {
+        console.error(err);
+      });
+    deleteCardModal.close();
```

Comment:
Needs correction: close the modal in `then` block after the successful api response

Reply:
Corrected!

Reply:
Corrected!